docs/modules/ROOT/pages/references/architecture/espejote-in-cluster-templating-controller.adoc

```diff
@@ -85,9 +98,49 @@ The template is re-rendered whenever the watched resource is created, changed, o
 The `minTimeBetweenTriggers` field can be used to limit the rate of re-renders.
 This is useful to prevent a high rate of re-renders when a resource is changed multiple times in a short period.
 If a list of resources is watched, the `minTimeBetweenTriggers` is applied to each resource individually.
+`minTimeBetweenTriggers` is https://github.com/vshn/espejote/issues/45[coming soon].
 
 The `watchResource` trigger contains the full Kubernetes resource definition of the watched resource.
 
+The resource is available in the Jsonnet template as `esp.triggerData().resource`.
+
+==== `watchContextResource`
+
+The `watchContextResource` trigger works the same as the `watchResource` trigger, but copies cache and configuration from a context resource.
```

Comment:
Maybe reword a bit to make it clearer that this will share the cache with the context?

```suggestion
The `watchContextResource` trigger works the same as the `watchResource` trigger, but shares cache and configuration with a context resource.
```